ports/espressif/common-hal/wifi/Radio.c

```diff
@@ -86,6 +86,9 @@ void common_hal_wifi_radio_set_enabled(wifi_radio_obj_t *self, bool enabled) {
     if (!self->started && enabled) {
         ESP_ERROR_CHECK(esp_wifi_start());
         self->started = true;
+        #if CIRCUITPY_REDUCE_TX_POWER
+        common_hal_wifi_radio_set_tx_power(self, 11.0);
+        #endif
```

Comment:
This is tricky. The C preprocessor can't dp `#if` with floating constants. So in order to conditionalize this so the `common_hal_wifi_radio_set_tx_power()` is not compiled if the power is not specified, we use a constant expression which will be optimized.
```suggestion
        if (CIRCUITPY_WIFI_DEFAULT_TX_POWER != 20) {
            common_hal_wifi_radio_set_tx_power(self, CIRCUITPY_WIFI_DEFAULT_TX_POWER);
        }
```